### PR TITLE
6267-6268

### PR DESCRIPTION
**Pull request by sealDebolina**
…larificaton_on13thSept

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the enable_keep_helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/applicant/add-applicants/content.test.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { form, generateContent, generateFormFields } from './content';
 jest.mock('../../../../app/form/validation');
 
 const en = {
-  pageTitle: 'Enter your name  ',
+  title: 'Enter your name  ',
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/applicant/add-applicants/content.test.ts:15]: remove space

File: src/main/steps/c100-rebuild/applicant/confidentiality/details-know/content.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { isFieldFilledIn } from '../../../../../app/form/validation';
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 export const en = () => ({
   caption: 'Keeping your contact details private for',
-  headingTitle: 'Do the other people named in this application (the respondents) know any of your contact details?    ',
+  title: 'Do the other people named in this application (the respondents) know any of your contact details?    ',
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/applicant/confidentiality/details-know/content.ts:9]: remove space

File: src/main/steps/c100-rebuild/hearing-urgency/urgent-details/template.njk [hunk 3]
```diff
@@ -3,5 +3,5 @@
 
 {% block page_content %}
 <h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{title}}</h1>
-<h1 class="govuk-heading-m">{{ reasonForUrgentHearing }}</h1>
+<h2 class="govuk-heading-m">{{ reasonForUrgentHearing }}</h1>
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/hearing-urgency/urgent-details/template.njk:6]: wrong closing tag

File: src/main/steps/c100-rebuild/help-with-fees/fees-applied/content.ts [hunk 4]
```diff
@@ -53,10 +53,13 @@ export const form: FormContent = {
           label: l => l.one,
           value: YesOrNo.YES,
           subFields: {
+            helpWithFeesReferenceLabel: {
+              type: 'textAndHtml',
+              textAndHtml: l =>
+                `<h3 class="govuk-heading-m">${l.hwfReferenceNumberLabel}</h3>${l.hwfReferenceNumberBody}`,
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/help-with-fees/fees-applied/content.ts:59]: check if space required or not.

File: src/main/steps/common/safety-concerns/child/template.njk [hunk 5]
```diff
@@ -2,13 +2,13 @@
...
 </p>
-<h1 class="govuk-heading-m">{{ reportIncident }}</h1>
+<h2 class="govuk-heading-m">{{ reportIncident }}</h2>
 <p class="govuk-body-m"> {{ emergencyCall }} </p>
 
 <p class="govuk-body-m">
 <a href="{{howToReportChildAbuseHyperLink}}" id="howToReportChildAbuseLabel" target="_blank" class="govuk-link">{{ howToReportChildAbuseLabel }}</a>
 </p>
-<h1 class="govuk-heading-m">{{ support }}</h1>
+<h2 class="govuk-heading-m">{{ support }}</h1>
```
Comment (vivek-sekhar) [inline src/main/steps/common/safety-concerns/child/template.njk:11]: wrong closing tag

File: src/main/steps/common/safety-concerns/no-feedback/template.njk [hunk 6]
```diff
@@ -3,7 +3,7 @@
 <span class="govuk-caption-l">{{caption}}</span>
 <h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{title}}</h1>
 <p class="govuk-body">{{firstParagraph}}</p>
-<h1 class="govuk-heading-m">{{subHeading}}</h1>
+<h2 class="govuk-heading-m">{{subHeading}}</h1>
```
Comment (vivek-sekhar) [inline src/main/steps/common/safety-concerns/no-feedback/template.njk:6]: wrong closing tag

File: src/main/steps/common/safety-concerns/yourself/template.njk [hunk 7]
```diff
@@ -3,13 +3,13 @@
...
  </p>
- <h1 class="govuk-heading-m">{{ reportIncident }}</h1>
+ <h2 class="govuk-heading-m">{{ reportIncident }}</h2>
  <p class="govuk-body-m"> {{ emergencyCall }} </p>
 
  <p class="govuk-body-m">
  <a href="{{reportDomesticAbuseHyperLink}}" id="reportDomesticAbuseLabel" target="_blank" class="govuk-link">{{ reportDomesticAbuseLabel }}</a>
  </p>
- <h1 class="govuk-heading-m">{{ support }}</h1>
+ <h2 class="govuk-heading-m">{{ support }}</h1>
```
Comment (vivek-sekhar) [inline src/main/steps/common/safety-concerns/yourself/template.njk:12]: wrong closing tag

File: src/main/steps/common/statement-of-service/review/template.njk [hunk 8]
```diff
@@ -20,7 +20,7 @@
         })
       }}
 
-  <h1 class="govuk-heading-m">{{ statementOfTruthHeading }}</h1>
+  <h2 class="govuk-heading-m">{{ statementOfTruthHeading }}</h1>
```
Comment (vivek-sekhar) [inline src/main/steps/common/statement-of-service/review/template.njk:23]: wrong closing tag

File: src/main/steps/common/yourhearings/hearings/template.njk [hunk 9]
```diff
@@ -14,7 +14,7 @@
    <p class="govuk-body"><a href={{linkforsupport}}>{{supportDuringCaselinktext}} </a></p>
    <p class="govuk-body"><a href={{linkfordelayorcancel}}>{{delayorcancellinktext}} </a></p>
    {% for item in nextHearing %}
-    <h1 class="govuk-heading-m">{{ nextHearingHeading }}</h1>
+    <h2 class="govuk-heading-m">{{ nextHearingHeading }}</h1>
```
Comment (vivek-sekhar) [inline src/main/steps/common/yourhearings/hearings/template.njk:17]: wrong closing tag

File: src/main/steps/complete-your-application-guidance/template.njk [hunk 10]
```diff
@@ -8,7 +8,7 @@
     <p class="govuk-body">{{paragraph | safe}}</p>
   {% endfor %}
 
-  <h1 class="govuk-heading-m">{{helpPayingCourtFeesSubHeading}}</h1>
+  <h2 class="govuk-heading-m">{{helpPayingCourtFeesSubHeading}}</h1>
```
Comment (vivek-sekhar) [inline src/main/steps/complete-your-application-guidance/template.njk:11]: wrong closing tag

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/help-with-fees/fees-applied/content.ts (hunk 4 above)
Comment (sealDebolina) [reply src/main/steps/c100-rebuild/help-with-fees/fees-applied/content.ts:59]: not needed

File: src/main/steps/application-within-proceedings/guidance/template.njk [hunk 11]
```diff
@@ -18,7 +18,7 @@
     <p class="govuk-body">{{content | safe}}</p>
   {% endfor %}
 
-  <h1 class="govuk-heading-m">{{helpWithFees.title}}</h1>
+  <h2 class="govuk-heading-m">{{helpWithFees.title}}</h1>
```
Comment (vivek-sekhar) [inline src/main/steps/application-within-proceedings/guidance/template.njk:21]: wrong closing tag

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**